doc/modules/model_evaluation.rst

```diff
@@ -100,7 +100,7 @@ Scoring                                Function
 'r2'                                   :func:`metrics.r2_score`
 'neg_mean_poisson_deviance'            :func:`metrics.mean_poisson_deviance`
 'neg_mean_gamma_deviance'              :func:`metrics.mean_gamma_deviance`
-'neg_mean_absolute_percentage_error'   :func:`metrics.mean_absolute_percentage_error`
+'neg_mean_absolute_percentage_error'   :func:`metrics.mean_absolute_percentage_error`     should not be used when there is actaul 0 in the data
```

Comment:
Going over the other comments in the table, they give more information about the predefined metric string and not "how to use the metric".

For docstrings about "how to use the metric", it's best to include them in the [API docs](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html) or the [User Guide](https://scikit-learn.org/stable/modules/model_evaluation.html#mean-absolute-percentage-error). The API docs we already have a statement about small `y_true`:

https://github.com/scikit-learn/scikit-learn/blob/c991e30c96ace1565604b429de22e36ed6b1e7bd/sklearn/metrics/_regression.py#L322-L325

I'm okay with expanding the user guide to state the behavior of small `y_true` values:

https://github.com/scikit-learn/scikit-learn/blob/c991e30c96ace1565604b429de22e36ed6b1e7bd/doc/modules/model_evaluation.rst?plain=1#L2345-L2346